src/content/docs/apis/nerdgraph/examples/nerdgraph-issues-api-via-github.mdx

```diff
@@ -0,0 +1,2103 @@
+redirects: 
+---
+
+This doc is a place for you and your team to learn how to query your data for issues and incidents and then configure features using our NerdGraph API.
+
+Our two main data types are `Issue Type` and `Incident Type`. 
+
+## Issue Type [#issue-type]
+
+You can learn more about New Relic's data types here:
+
+<table>
+  <thead>
+    <tr>
+      <th style={{ width: "200px" }}>
+        Name
+      </th>
+
+      <th>
+        Type
+      </th>
+
+      <th>
+        Description
+      </th>
+    </tr>
+  </thead>
+
+  <tbody>
+     <tr>
+      <td>
+        accountIds
+      </td>
+
+      <td>
+        [Int!]!
+      </td>
+
+      <td>
+       A list of account Ids. As an issue could be correlated with others from other accounts.
+      </td>
+    </tr>
+    <tr>
+      <td>
+        acknowledgedAt
+      </td>
+
+      <td>
+        EpochMilliseconds
+      </td>
+
+      <td>
+       The time the issue was acknowledged.
+      </td>
+    </tr>
+    <tr>
+      <td>
+        acknowledgedBy
+      </td>
+
+      <td>
+        String
+      </td>
+
+      <td>
+        The user that acknowledged the issue.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        activatedAt
+      </td>
+
+      <td>
+        EpochMilliseconds
+      </td>
+
+      <td>
+        The time the issue was activated: createdAt + gracePeriod.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        closedAt
+      </td>
+
+      <td>
+        EpochMilliseconds
+      </td>
+
+      <td>
+        The time the issue was closed.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        closedBy
+      </td>
+
+      <td>
+        String
+      </td>
+
+      <td>
+        The user that manually closed the issue.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        createdAt
+      </td>
+
+      <td>
+        EpochMilliseconds
+      </td>
+
+      <td>
+        The time the issue was created.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        description
+      </td>
+
+      <td>
+        [String!]!
+      </td>
+
+      <td>
+        A list of incident descriptions.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        entityGuids
+      </td>
+
+      <td>
+        [EntityGuid!]
+      </td>
+
+      <td>
+        A list of unique entity identifiers.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        entityNames
+      </td>
+
+      <td>
+        [String!]
+      </td>
+
+      <td>
+        A list of entity names.
+      </td>
+    </tr>
+
+
+    <tr>
+      <td>
+        entityTypes
+      </td>
+
+      <td>
+       [String!]
+      </td>
+
+      <td>
+        A list of entity types.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        environmentId
+      </td>
+
+      <td>
+        [Int!]!
+      </td>
+
+      <td>
+        See [our documentation](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/get-started-incident-intelligence/) to learn about environments.
```

Comment:
according to style guide, doc link text should be specific about what it is. so it'd be something like See [our docs on environments](...)